<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWin</class>
 <widget class="QWidget" name="MyWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWin</string>
  </property>
  <widget class="QPushButton" name="btnArrow">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountCmBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistAccount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Register Now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnForgotPasswd">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Forgot Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoreAccountLogin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFlicker">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="keepPasswdCmBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Keep Password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoLoginCmBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoginState">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="userHead">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>82</width>
     <height>82</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>userHead</zorder>
  <zorder>btnArrow</zorder>
  <zorder>btnLogin</zorder>
  <zorder>accountCmBox</zorder>
  <zorder>passwdLineEdit</zorder>
  <zorder>btnRegistAccount</zorder>
  <zorder>btnForgotPasswd</zorder>
  <zorder>btnMoreAccountLogin</zorder>
  <zorder>btnFlicker</zorder>
  <zorder>keepPasswdCmBox</zorder>
  <zorder>autoLoginCmBox</zorder>
  <zorder>btnLoginState</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
